doc/api.rst

```diff
@@ -862,6 +862,289 @@ Find the last 10 commands signed by an investigator identified by name.
+                "items": [
+                    {
+                        "data": [
+                            {
+                                "name": "manifest",
+                                "value": {
+                                    "entries": [
+                                        {
+                                            "name": "mig-loader",
+                                            "sha256": "<object sha256sum...>"
+                                        },
+                                        {
+                                            "name": "configuration",
+                                            "sha256": "<object sha256sum...>"
+                                        },
+                                        {
+                                            "name": "mig-agent",
+                                            "sha256": "<object sha256sum...>"
+                                        },
+                                        {
+                                            "name": "agentcert",
+                                            "sha256": "<object sha256sum...>"
+                                        },
+                                        {
+                                            "name": "cacert",
+                                            "sha256": "<object sha256sum...>"
+                                        },
+                                        {
+                                            "name": "agentkey",
+                                            "sha256": "<object sha256sum...>"
+                                        }
+                                    ],
+                                    "loader_name": "khan",
+                                    "signatures": [
+                                        "<a signature from a MIG administrator...>"
+                                    ]
+                                }
+                            }
+                        ],
+                        "href": "/api/v1/manifest/agent/"
+                    }
+                ],
+                "template": {},
+                "version": "1.0"
+            }
+        }
+
+POST /api/v1/manifest/fetch/
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Description: Fetches a file provided by a manifest
+* Parameters: (POST body)
+	- `parameters`: JSON marshaled mig.ManifestParameters data
+* Authentication: X-LOADERKEY
+* Response Code: 200 OK
+* Reponse: Collection+JSON
+
+.. code:: json
+
+        {
+            "collection": {
+                "error": {},
+                "href": "http://api.mig.example.net:1664/api/v1/manifest/fetch/",
+                "items": [
+                    {
+                        "data": [
+                            {
+                                "name": "content",
+                                "value": {
+                                    "data": "<file content...>",
```

Comment:
Could we specify the encoding of the file content? I think it's base64, it is compressed?